cypress/e2e/po/components/sortable-table.po.ts

```diff
@@ -56,6 +56,10 @@ export default class SortableTablePo extends ComponentPo {
     return cy.getId('sortable-table-promptRemove');
   }
 
+  downloadYamlButton() {
```

Comment:
missed this. yaml download will be something specific to a generic resource list rather than the core sortable table

Reply:
@richard-cox so should it live in `/po/components/resource-table.po.ts`?

Reply:
I thought so, and checked, there's already a downloadYamlButton in there that can be used

Reply:
oh okay, I moved it from `resource-table.po` to `sortable-table.po` in this commit https://github.com/rancher/dashboard/pull/10161/commits/805d37f7aba0835eb5200b3ee0c3b42a198ea300 but I can move it back though